python/ray/worker.py

```diff
@@ -1516,22 +1519,43 @@ def get(object_refs, *, timeout=None):
 
 
 @client_mode_hook
-def put(value):
+def put(value, *, _owner=None):
     """Store an object in the object store.
 
     The object may not be evicted while a reference to the returned ID exists.
 
     Args:
         value: The Python object to be stored.
+        _owner: The `ActorHandle` or serialized address of object's owner.
```

Comment:
Can we only support actor handles?

Reply:
```suggestion
        _owner: The actor that should own this object. This allows creating objects
           with lifetimes decoupled from that of the creating process. Note that the
           owner actor must be passed a reference to the object prior to the object
           creator exiting, otherwise the reference will still be lost.
```

Reply:
It seems that no one uses the serialization address, and I will delete it later. By the way, thank you for the modification of the Annotation